src/util/file_log.rs

```diff
@@ -42,6 +42,11 @@ fn compute_rollover_time(tm: Tm) -> Tm {
     (day_start_tm.to_utc() + duration).to_local()
 }
 
+fn one_day_before(tm: Tm) -> Tm {
```

Comment:
can add a test for this. E,g, construct a Tm, like 08-30, the one_day_before will be 08-29.